tests/php/Controllers/ContentControllerTest.php

```diff
@@ -204,7 +205,7 @@ public function testGetViewer()
                 __DIR__
                 . '/themes/controllertest/templates/SilverStripe/CMS/Tests/Controllers/'
                 . 'ContentControllerTestPage_test.ss',
-                $viewer->templates()['main']
+                Deprecation::withSuppressedNotice(fn() => $viewer->templates())['main']
```

Comment:
Deprecation added in https://github.com/silverstripe/silverstripe-framework/pull/11420